packages/calcite-components/src/components/popover/popover.e2e.ts

```diff
@@ -699,4 +701,67 @@ describe("calcite-popover", () => {
       });
     });
   });
+
+  describe("theme", () => {
+    describe("default", () => {
+      themed(
+        html`
+          <calcite-popover heading="I'm a heading in the header using the 'heading' prop!">
+            Lorem Ipsum
+          </calcite-popover>
+        `,
+        {
+          "--calcite-popover-background-color": {
+            shadowSelector: `.${FloatingCSS.animation}`,
```

Comment:
Sidebar: for consistency, can we add a `.container` class to this element and use that in the test instead?

Reply:
Oh, I didn't spot that. Thanks for clarifying. We should look at renaming the lower-level one since it's inconsistent with the way we use `.container` in other components.